packages/cli/src/commands/profile/downloadProfile.ts

```diff
@@ -64,6 +67,83 @@ function validatePackageName(packageName: string) {
   return /^[a-z][a-z0-9_]*(\.[a-z][a-z0-9_]*)+$/.test(packageName);
 }
 
+async function getSourcemapFromServer(): Promise<AxiosResponse<SourceMap>> {
+  const DEBUG_SERVER_PORT = '8081';
+  const IP_ADDRESS = ip.address();
+  const PLATFORM = 'android';
+  const requestURL = `http://${IP_ADDRESS}:${DEBUG_SERVER_PORT}/index.map?platform=${PLATFORM}&dev=true`;
+  return (await axios.get(requestURL)) as AxiosResponse<SourceMap>;
```

Comment:
I don't think we need axios for this. Taking the dependency for a single http request is most likely not going to be popular among the react-native-cli maintainers